interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: Calling a super in a class brings down the instances of the super class down the inheritance of the sub-classes. 
```

Comment:
I would really like to see you expand these answers in the future.  Currently, this is really meeting the bare minimum requirements of this section.  Make sure to provide _detailed_ definitions, real-life experiences with the concept, and/or examples to display a deeper level of understanding.